zebra/zebra_memory.c

```diff
@@ -27,6 +27,8 @@
 
 DEFINE_MGROUP(ZEBRA, "zebra")
 DEFINE_MTYPE(ZEBRA, RTADV_PREFIX, "Router Advertisement Prefix")
+DEFINE_MTYPE(ZEBRA, RTADV_RDNSS, "Router Advertisement RDNSS")
+DEFINE_MTYPE(ZEBRA, RTADV_DNSSL, "Router Advertisement DNSSL")
```

Comment:
These are only in `zebra_memory.c` for historical reasons (because noone has done a cleanup yet.)  Can you please move your 2 new lines to `zebra/rtadv.c` and use `DEFINE_MTYPE_STATIC` instead? (And remove the `DECLARE_MTYPE` in `zebra_memory.h`)

Reply:
Makes sense. Updated the PR.